### ISSUE TITLE
Can this be integrated into an existing Django project?

### ISSUE THREAD

**Issue by travis-shippingtree** [nitely/Spirit#240]
If so, can you provide instructions?

**Comment by nitely** [comment]
Yes and not really. Just follow the install steps, once you have your Spirit forum running, copy the settings into your project, merging them with whatever settings your project has. Then add the urls into urls.py. Run migrations. That's it. I can't provide instructions 'cause it depends on your project.
